app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_func = {}
+    def wrapper(*args, **kwargs) -> str:
```

Comment:
The return type annotation of the `wrapper` function is set to `str`, which may not be appropriate for all functions that this decorator could be applied to. Consider changing the return type to `Any` to accommodate functions that return different types.